extends Control
@onready var click_sound = preload("res://sonidos/botÃ³n2.wav")   # ðŸ”Š sonido click

func _ready():
	# Conectar botones
	$flecha_der.pressed.connect(_on_flecha_der_pressed)
	$flecha_izq.pressed.connect(_on_flecha_izq_pressed)
	$salir.pressed.connect(_on_salir_pressed)

func _input(event):
	# Navegar con teclado
	if event.is_action_pressed("ui_right"):  # tecla Flecha Derecha
		get_tree().change_scene_to_file("res://scenes/coleccionista3.tscn")
	if event.is_action_pressed("ui_left"):  # tecla Flecha Derecha
		get_tree().change_scene_to_file("res://scenes/coleccionista.tscn")
	elif event.is_action_pressed("ui_cancel"):  # tecla Esc
		get_tree().change_scene_to_file("res://scenes/menu_principal.tscn")

# Funciones de botones
func _on_flecha_der_pressed():
	_play_click()
	get_tree().change_scene_to_file("res://scenes/coleccionista3.tscn")
func _on_flecha_izq_pressed():
	_play_click()
	get_tree().change_scene_to_file("res://scenes/coleccionista.tscn")

func _on_salir_pressed():
	_play_click()
	get_tree().change_scene_to_file("res://scenes/menu_principal.tscn")

# --- ðŸ”Š Reproducir sonidos ---
func _play_click():
	AudioManager.play_click(click_sound)
